__tests__/integration/mirador/invalid-api-response.test.js

```diff
@@ -14,4 +14,25 @@ describe('Mirador Invalid API Response Handler Test', () => {
       expect(e.name).toMatch('TimeoutError');
```

Comment:
Not really part of my PR, but I puzzled over this for a bit.  I don't think this test will ever fail, because all this does is test that there is not an `li` w/ that text in the page (which there isn't, so, yeah, it's going to pass).

I'm not sure what the intent of the test is based on the "breaks Mirador" description, so I'm not really sure what to do with this.